resources/benchmark/jcch/customer.csv.json

```diff
@@ -0,0 +1,54 @@
+{
+    "columns": [
+        {
+            "name": "c_custkey",
+            "nullable": false,
+            "type": "int"
+        },
+        {
+            "name": "c_name",
+            "nullable": false,
+            "type": "string"
+        },
+        {
+            "name": "c_address",
+            "nullable": false,
+            "type": "string"
+        },
+        {
+            "name": "c_nationkey",
+            "nullable": false,
+            "type": "int"
+        },
+        {
+            "name": "c_phone",
+            "nullable": false,
+            "type": "string"
+        },
+        {
+            "name": "c_acctbal",
+            "nullable": false,
+            "type": "float"
+        },
+        {
+            "name": "c_mktsegment",
+            "nullable": false,
+            "type": "string"
+        },
+        {
+            "name": "c_comment",
+            "nullable": false,
+            "type": "string"
+        }
+    ],
+    "config": {
+        "delimiter": "\n",
+        "delimiter_escape": "\\",
+        "escape": "\"",
+        "null_handling": "reject_null_strings",
+        "quote": "\"",
+        "reject_quoted_nonstrings": true,
+        "rfc_mode": true,
+        "separator": "|"
```

Comment:
Generated by creating the TPC-H data, exporting it as CSV, and changing the separator from `,` to `|`.